dev/scripts/create_sources.py

```diff
@@ -70,8 +71,13 @@ def __init__(self, auth_header):
         header = {"x-rh-identity": auth_header}
         self._identity_header = header
 
-    def create_s3_bucket(self, parameters, billing_source):
-        json_data = {"billing_source": {"bucket": billing_source}}
+    def create_s3_bucket(self, parameters, billing_source, s3_region=None):
+        json_data = {
+            "billing_source": {
+                "bucket": billing_source,
+                "bucket_region": s3_region,
```

Comment:
Should this only be added if `s3_region` was provided?

Reply:
Given that this is just a dev script, I think it's fine to put it here. I'm not sure anyone uses this script anyway 😅

Reply:
I was wondering how much use this gets too. It looks like `create_test_customer.py`  and `load_test_customer_data.sh` do most of the work these days.